generated/nitclk/session.py

```diff
@@ -0,0 +1,1156 @@
+        return
+
+    def finish_sync_pulse_sender_synchronize(self, sessions, min_time):
+        r'''finish_sync_pulse_sender_synchronize
+
+        TBD
+
+        Args:
+            sessions (list of int): sessions is an array of sessions that are being synchronized.
+
+            min_time (float): Minimal period of TClk, expressed in seconds. Supported values are
+                between 0.0 s and 0.050 s (50 ms). Minimal period for a single
+                chassis/PC is 200 ns. If the specified value is less than 200 ns,
+                NI-TClk automatically coerces minTime to 200 ns. For multichassis
+                synchronization, adjust this value to account for propagation delays
+                through the various devices and cables.
+
+        '''
+        session_count_ctype = _visatype.ViUInt32(0 if sessions is None else len(sessions))  # case S160
+        sessions_ctype = get_ctypes_pointer_for_buffer(value=_converters.convert_to_nitclk_session_num_list(sessions), library_type=_visatype.ViSession)  # case B520
+        min_time_ctype = _visatype.ViReal64(min_time)  # case S150
+        error_code = self._library.niTClk_FinishSyncPulseSenderSynchronize(session_count_ctype, sessions_ctype, min_time_ctype)
+        errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
+        return
+
+    def _get_extended_error_info(self):
+        r'''_get_extended_error_info
+
+        Reports extended error information for the most recent NI-TClk method
+        that returned an error. To establish the method that returned an
+        error, use the return values of the individual methods because once
+        _get_extended_error_info reports an errorString, it does not report
+        an empty string again.
+
+        Returns:
+            error_string (str): Extended error description. If errorString is NULL, then it is not large
+                enough to hold the entire error description. In this case, the return
+                value of _get_extended_error_info is the size that you should use
+                for _get_extended_error_info to return the full error string.
+
+        '''
+        error_string_ctype = None  # case C050
+        error_string_size_ctype = _visatype.ViUInt32()  # case S170
+        error_code = self._library.niTClk_GetExtendedErrorInfo(error_string_ctype, error_string_size_ctype)
+        errors.handle_error(self, error_code, ignore_warnings=True, is_error_handling=True)
+        error_string_size_ctype = _visatype.ViUInt32(error_code)  # case S180
+        error_string_ctype = (_visatype.ViChar * error_string_size_ctype.value)()  # case C060
+        error_code = self._library.niTClk_GetExtendedErrorInfo(error_string_ctype, error_string_size_ctype)
+        errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=True)
+        return error_string_ctype.value.decode(self._encoding)
+
+    def init_for_documentation(self):
```

Comment:
?

Reply:
Removed in a later PR - not needed any more.